node/tests/common.rs

```diff
@@ -126,6 +126,25 @@ pub async fn get_treasury_balance(url: &str) -> Result<u128, Box<dyn std::error:
 	Ok(free_balance)
 }
 
+/// get qWFeXVApgApnQCtqEKURfvRJUpvUA22bLiEyEA2iapF4vcuqS next session key
+pub async fn get_next_session_keys(url: &str) -> Result<Vec<u8>, Box<dyn std::error::Error>> {
```

Comment:
probably could be more useful if we could specify keys, because currently it's hardcode for one test.
As far as I understand storage key.

Reply:
do you means add a address param to generate storage key？

Reply:
Yeah, but it's probably not worth to change right now